site/content/cluster/kubernetes_cluster.md

```diff
@@ -0,0 +1,95 @@
+---
+linktitle: "Kubernetes"
+weight: 1
+---
+
+# Creating an M3 Cluster with Kubernetes
+
+This guide shows you how to create an M3 cluster of 3 nodes, designed to run locally on the same machine. It is designed to show you how M3 and Kubernetes can work together, but not as a production example.
+
+{{% notice note %}}
+This guide assumes you have read the [quickstart](/docs/quickstart), and builds upon the concepts in that guide.
+{{% /notice %}}
+
+{{% notice tip %}}
+We recommend you use [our Kubernetes operator](https://operator.m3db.io/) to deploy M3 to a cluster. It is a more streamlined setup that uses [custom resource definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) to automatically handle operations such as managing cluster placements.
+{{% /notice %}}
+
+
+## Prerequisites
+
+-   A running Kubernetes cluster.
+    -   For local testing, you can use [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/), [Docker desktop](https://www.docker.com/products/docker-desktop), or [we have a script](https://raw.githubusercontent.com/m3db/m3db-operator/master/scripts/kind-create-cluster.sh) you can use to start a 3 node cluster with [Kind](https://kind.sigs.k8s.io/docs/user/quick-start/).
+-   To use the M3DB operator chart, you need [Helm](https://helm.sh/)
+
+## Create An etcd Cluster
+
+M3 stores its cluster placements and runtime metadata in [etcd](https://etcd.io) and needs a running cluster to communicate with.
+
+We have example services and stateful sets you can use, but feel free to use your own configuration and change any later instructions accordingly.
+
+```shell
+kubectl apply -f https://github.com/m3db/m3db-operator/blob/v0.9.0/example/etcd/etcd-minikube.yaml
```

Comment:
This file no longer exists, can we link to this instead?
https://github.com/m3db/m3db-operator/blob/master/example/etcd/etcd-basic.yaml

Reply:
This is again one of mine waiting on… https://github.com/m3db/m3db-operator/pull/244 it has a lot of tweaks to make a local installation work, I don't think I could get _etcd-basic_ to work.